blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/outputs/handlers/HttpRollupHandlerWithESIntegrationTest.java

```diff
@@ -135,7 +135,7 @@ public void testUnknownUnit() throws Exception {
                 baseMillis + 86400000,
                 1600);
         Assert.assertEquals(data.getData().getPoints().size(), 0);
-        Assert.assertEquals(data.getUnit(), "unknown");
+        Assert.assertNull(data.getUnit());
```

Comment:
@chinmay-gupte I'd like it, if we do spend a little more time on this because we do Util.Unknown in the last layer that is serialization. If we are sending nulls to Serialization because of a bug, this test should be able to catch it. What do you think?